### ISSUE TITLE
During setup, Mainnet data is displayed for Testnet txn

### ISSUE THREAD

**Issue by bitcoin333** [randlabs/myalgo-connect#34]
When selecting the accounts for signing, the Mainnet balances are shown, even when the txns to sign are Testnet txns.

**Comment by Alex99y** [comment]
Hello @bitcoin333 ! Do you mean during the `myAlgo.connect()` phase?

**Comment by bitcoin333** [comment]
Yes. Can I specify somehow the genesisHash and genesisID during the call?

**Comment by Alex99y** [comment]
@bitcoin333  Currently it is not possible. The connect operation is independent of the network
